client/components/rich_text/components/input_url.jsx

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import colors from '@artsy/reaction-force/dist/Assets/Colors'
+import { RemoveButton } from 'client/components/remove_button'
+
+export class TextInputUrl extends Component {
+  static propTypes = {
+    confirmLink: PropTypes.func,
+    pluginType: PropTypes.string,
+    removeLink: PropTypes.func,
+    selectionTarget: PropTypes.object,
+    urlValue: PropTypes.string
+  }
+
+  state = {
+    url: this.props.urlValue || ''
+  }
+
+  confirmLink = () => {
+    const { url } = this.state
+    const {
+      confirmLink,
+      pluginType,
+      removeLink
+    } = this.props
+
+    if (url.length) {
+      confirmLink(url, pluginType)
+    } else {
+      removeLink()
+    }
+  }
+
+  render () {
+    const { url } = this.state
+    const { removeLink, selectionTarget } = this.props
+    const { left, top } = selectionTarget
+
+    return (
+      <div
+        className='TextInputUrl'
+        style={{
+          top: top || 0,
+          left: left || 0
+        }}
```

Comment:
Possible to choose styled-components or CSS on this? I think we've discussed before but it makes it a little hard to track down when we're using 3 different ways to style.

Reply:
Good idea - will add this.